gallery/src/components/NxTabs/NxTabsSimpleExample.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel
+} from '@sonatype/react-shared-components';
+
+const NxTableSimpleExample = () => {
+  const [activeTabId, setActiveTabId] = useState('tab-1');
+
+  function handleKeyPress(event: React.KeyboardEvent<HTMLLIElement>) {
```

Comment:
Requiring calling code to set up keyboard handlers on every tab isn't really alright.  Keyboard nav needs to be built into the component.  As I said before, I think the cleanest way to get browser-default behavior would be to build it around radio buttons.

Reply:
I think I'm going to leave the radio button implementation for another day. `NxTab` now sets up it's own keyboard handler internally that hooks to the `onTabSelect` event. This works fine in my quick testing, but we may want to beef it up later.